internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/consumer/ApacheKafkaConsumerAdapter.java

```diff
@@ -468,7 +475,13 @@ public Long endOffset(PubSubTopicPartition pubSubTopicPartition) {
    */
   @Override
   public List<PubSubTopicPartitionInfo> partitionsFor(PubSubTopic topic) {
-    List<PartitionInfo> partitionInfos = this.kafkaConsumer.partitionsFor(topic.getName());
+    List<PartitionInfo> partitionInfos = null;
+    try {
+      partitionInfos = this.kafkaConsumer.partitionsFor(topic.getName());
+    } catch (Exception e) {
+      LOGGER.error("Exception thrown when attempting to get partitions for topic: {}", topic, e);
+    }
```

Comment:
I don't want us to swallow InterruptException; can we capture Kafka exception only?

Reply:
good point